src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -80,6 +80,7 @@ public enum ByteCode :byte
         STOREEXIST     = 0x5c,
         PUSHDELEGATE   = 0x5d,
         BRANCHTRUE     = 0x5e,
+        ARGBOTTOM      = 0x5f,
```

Comment:
Warning: This will merge-conflict with branch "fixes_1092_defined_keyword".  This is because they both added a new Opcode, and both gave it code number ``0x5f``.

That was because I had to make them as separate independant pull requests from the same starting develop branch, and in each one I added just the one opcode.

@erendrake, when merging them, just give one of them the code of ``0x5f`` while you give the other one code ``0x60`` - it doesn't matter which one you do which with.